src/theme.js

```diff
@@ -18,20 +18,26 @@ const theme = createMuiTheme({
     },
   },
   typography: {
-    useNextVariants: true,
     ...Roboto400,
-    display4: Roboto300,
-    display3: Roboto400,
-    display2: Roboto400,
-    display1: Roboto400,
-    headline: Roboto400,
-    title: Roboto500,
-    subheading: Roboto400,
-    body2: Roboto500,
-    body1: Roboto400,
+    h1: Roboto300,
+    h2: Roboto400,
+    h3: Roboto400,
+    h4: Roboto400,
+    h5: Roboto400,
+    h6: Roboto500,
+    subtitle1: Roboto400,
+    body1: Roboto500,
+    body2: Roboto400,
     caption: Roboto400,
     button: Roboto500,
   },
+  overrides: {
+    MuiButton: {
+      root: {
+        lineHeight: 1.5,
+      },
+    },
+  },
```

Comment:
We can remove the MuiButton override. I know they changed it to 1.75 but that's okay.